### DISCUSSION TITLE
bird detections not working

### DISCUSSION THREAD

**Discussion by Birdsphotos** [Nachtzuster/BirdNET-Pi#215] [Q&A]
help a noob-
I am struggling to get this working on an old X86 machine.
installed Debian 12 (Bookworm)
Log file seems to be blank:
![log](https://github.com/user-attachments/assets/b831a2d7-c58e-4367-bd0b-37711463b4f7)
![services](https://github.com/user-attachments/assets/2346b692-3519-4afa-8330-a6589a155aed)
![status](https://github.com/user-attachments/assets/4e805343-cd97-4f07-84be-d1fefd42869f)
Every thing seems to be running  
The txt file in
![analysing now ](https://github.com/user-attachments/assets/879ed35e-aeea-44db-8f19-e006f86e7ff5)
 "/BirdSongs/StreamData/analyzing_now.txt" does not seem to change and the Processed folder ins empty

**Comment by unosonic** [comment]
Requirements: A Raspberry Pi 5, Raspberry 4B, Raspberry Pi 400, Raspberry Pi 3B+, or Raspberry Pi 0W2 (The 3B+ and 0W2 must run on RaspiOS-ARM64-Lite) 
Maybe I've missed something, but I don't see an x86 option, so I wouldn't expect it to run on anything else then ARM architectures, at least not out of the box. 
edit: seems there are options, see https://github.com/Nachtzuster/BirdNET-Pi/discussions/13

**Comment by alexbelgium** [comment]
Clearly you were able to install it and start so your system supports it.

What does your spectrogram looks like? If your wav files pile up it is indeed very probably an issue with the birdnet analysis service- as the recording service seems to work fine. Are you sure there is no logs, you've waited a bit? Perhaps you should re-run the installer in case the services did not correctly install...

**Reply by Birdsphotos** [reply]
![spec](https://github.com/user-attachments/assets/ae9297cf-e5d8-4d2e-b14c-7ae0553e1fd3)
Here is a screenshot of the Spectrogram taken while I could hear a bird calling outside.
There is another tread on X86 install that I was pointed to after posting this. 
I have reinstalled it and also tried this mention here: #13 
`su birdnet
cd
source BirdNET-Pi/birdnet/bin/activate
pip3 uninstall tflite_runtime
pip3 install tensorflow
deactivate
sudo reboot`

**Reply by alexbelgium** [reply]
Looks quite good ! You could use alsamixer to increase the volume a bit (https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card) but it's quite clean.

The logs would really be the most helpful in troubleshooting however... Is there still nothing in View log? You could perhaps try to restart the recording service and birdnet_analysis server from the Services page and check if nothing changes in the logs?

**Comment by Birdsphotos** [comment]
decided to install the app on top of ubuntu instead of debian and everything works
